### ISSUE TITLE
Throws '[vue-async-data] not installed!' warning when attached as mixin 

### ISSUE THREAD

**Issue by iJackUA** [vuejs/vue-async-data#3]
Because of `install` method is not called in mixin (called only in `use` for plugins)

But according to docs
```
// use globally
// you can also just use `VueAsyncData.mixin` where needed
Vue.use(VueAsyncData)
```
it can be assigned separately to components via `mixins: [VueAsyncData.mixin],`

**Comment by yyx990803** [comment]
Did you call `Vue.use()`? You still need to do that even if using as a mixin.

**Comment by iJackUA** [comment]
No, I do not call `Vue.use()` in that case (with Vue 0.12.* it was working for sure without global `Vue.use(VueAsyncData) , but with a mixins in certain components).
But if to call `Vue.use(VueAsyncData)` `asyncData` became available in every Component. Why do I need to do `mixins: [VueAsyncData.mixin]` on any specific component then?

Why I actually started to play with `mixins` - as I saw in debugger that this mixing is being checked on the whole bunch of components on the page (also on every `v-repeat`) - and I was afraid of performance penalty.

**Comment by yyx990803** [comment]
Ah you're right. I'll fix that in a later alpha release.

**Comment by yyx990803** [comment]
Should be fixed in 1.0.2

**Comment by tomasz-szymanek** [comment]
I came across same thing using vue-async-data. I'm using mixin, and warning still is diplaying.
